client/src/components/App/App.js

```diff
@@ -1,13 +1,8 @@
 import './App.css';
 import { Routes } from '../../routes/routes';
 
-
-function App() {
-  return (
-    <>
-      <Routes/>
-    </>
-  );
-}
+const App = function () {
```

Comment:
App should be an arrow function.
```suggestion
const App = () => {
```